src/SampleLinter/Rules/NoGroupsLinterRule.cs

```diff
@@ -0,0 +1,101 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Dynamo.Graph.Nodes;
+using Dynamo.Graph.Workspaces;
+using Dynamo.Linting.Interfaces;
+using Dynamo.Linting.Rules;
+
+namespace SampleLinter.Rules
+{
```

Comment:
Let's add the rules description from your pr head on top of each rule as a summary comment.